lib/geokit/geocoders/openstreetmap.rb

```diff
@@ -110,4 +110,6 @@ def self.set_bounds(result_json, loc)
       end
     end
   end
+
+  Geokit::Geocoders::OsmGeocoder = Geokit::Geocoders::OSMGeocoder
```

Comment:
Why did you do this. I feel we're just adding another alias when people are already using OSM, if anything the class should reflect the file name (or file name reflect the class, e.g. OpenStreetMapGeocoder